roles/common/templates/monitoring/config.json

```diff
@@ -6,7 +6,11 @@
       "openstack"
     ],
     "keepalive": {
-      "handler": "{{ monitoring.keepalive.handler }}"
+      "handler": "{{ monitoring.keepalive.handler }}",
+      "thresholds": {
+        "warning": {{ monitoring.keepalive.thresholds.warning }},
```

Comment:
dont we need to wrap this around double quotes? same comment for below line as well. "{{ monitoring.keepalive.thresholds.warning }}"